build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+rm build -rf;
+mkdir build;
+cd build;
+qmake -qt=qt5 ../TFTrue.pro -r -spec linux-g++;
+make;
+sync;
+cp TFTrue.so /home/steph/tfTEST/tf2/tf/addons/TFTrue.so
```

Comment:
I think that file was not supposed to be included

Reply:
oops